pom.xml

```diff
@@ -308,6 +309,7 @@
     <surefire.rerunFailingTestsCount>0</surefire.rerunFailingTestsCount>
     <swissbox.version>1.8.3</swissbox.version>
     <testcontainers.version>1.19.7</testcontainers.version>
+    <testng.version>7.10.2</testng.version>
```

Comment:
testng dependency declared in dptMngt doesn't look to be used so remove it

Reply:
@olamy it is force to use latest testng version since reactive-streams-tck uses testng 7.3.0 version that introduce the following problems:
1. Unnecessary dependencies: junit 4.12, guice 4.2.2, ant 1.19 and snakeyaml 1.21. In latest testng 7.10.2 version there is no dependencies at all.
2. Introduce vulnerabilities. I think you have security checks even in the test classpath
3. Classpath collision between hamcrest-core 1.3 and hamcrest 2.2 and build will fail

Reply:
@scrat98 good point. this makes sense.